docs/ref/class-based-views/generic-date-based.txt

```diff
@@ -254,19 +254,19 @@ views for displaying drilldown pages for date-based data.
 
     **Example myapp/urls.py**::
 
-        from django.conf.urls import url
+        from django.urls import path
 
         from myapp.views import ArticleMonthArchiveView
 
         urlpatterns = [
-            # Example: /2012/aug/
-            url(r'^(?P<year>[0-9]{4})/(?P<month>[-\w]+)/$',
-                ArticleMonthArchiveView.as_view(),
-                name="archive_month"),
             # Example: /2012/08/
-            url(r'^(?P<year>[0-9]{4})/(?P<month>[0-9]+)/$',
+            path('<int:year>/<int:month>/',
                 ArticleMonthArchiveView.as_view(month_format='%m'),
                 name="archive_month_numeric"),
+            # Example: /2012/aug/
+            path('<int:year>/<str:month>/',
```

Comment:
The string converter is currently registered with "string" rather than "str". Do you think we should change code or docs? "str" seems more consistent with "int" to me.

Reply:
I'd suppose that we might as well keep consistency with werkzeug/flask. (ie. use "string")

Reply:
I wonder if `string` was chosen by werkzeug/flask in a Python 2/3 to disambiguate the whole unicode/str situation. I also prefer `str` given Django is only supporting Python 3 now.

Reply:
@charettes Good point.